unity-renderer/Assets/DCLPlugins/CameraReelPlugin/CameraReelPlugin.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Cysharp.Threading.Tasks;
+using DCL;
+using DCL.Providers;
+using DCLFeatures.CameraReel.Section;
+using UnityEngine;
+
+namespace DCLPlugins.CameraReelPlugin
+{
+    public class CameraReelPlugin : IPlugin
+    {
+        private const string ADDRESS = "CameraReelSectionView";
+
+        private Transform sectionParent;
+        private CameraReelSectionController controller;
+
+        public CameraReelPlugin()
+        {
+            Initialize().Forget();
+        }
+
+        private async UniTaskVoid Initialize()
+        {
+            IAddressableResourceProvider assetProvider = Environment.i.platform.serviceLocator.Get<IAddressableResourceProvider>();
+
+            CameraReelSectionView view = await CreateCameraReelSectionView(assetProvider);
```

Comment:
Shouldn't we still have a cancellation token to dispose in the Dispose method? What happens if we stop the execution in unity before this await is completed?